handbook/engineering/leadership/index.md

```diff
@@ -43,33 +43,33 @@ We have a 1hr sync every Monday at 9:30am PT.
 - This meeting is recorded and automatically posted to #eng-manager-team.
 - Our [meeting notes](https://docs.google.com/document/d/1nZTGua3cdMTJLwANlBZNX2uXiDXLuYDBkH9tehwdqTY/edit#) are also internally available to the whole team.
 
-### Weekly updates
+### Weekly highlight
```

Comment:
I believe we are missing here some info about highlights not being mandatory every week.